src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -343,9 +343,12 @@ class EmojiPickerMenu extends Component {
             return;
         }
 
-        const newFilteredEmojiList = _.filter(this.emojis, emoji => (
+        // We're skipping "Frequently Used" emojis to avoid duplicate results. Frequently Usd Emoji is the 0th index category.
+        // Hence, we slice the array from the 1st index category onwards
+        const uniqueEmojis = _.isEmpty(this.props.frequentlyUsedEmojis) ? this.emojis : this.emojis.slice(this.unfilteredHeaderIndices[1] * this.numColumns);
```

Comment:
I am little confused about `this.unfilteredHeaderIndices[1] ` where did it come from?

Reply:
https://github.com/Expensify/App/blob/dcd288224f59bf686c5f8a472c30c4c15a095486/src/pages/home/report/EmojiPickerMenu/index.js#L70-L73

We're initializing `unfilteredHeaderIndices` in the above line. These `allEmojis` are returned from `EmojiUtils` as highlighted in the following block.

https://github.com/Expensify/App/blob/dcd288224f59bf686c5f8a472c30c4c15a095486/src/libs/EmojiUtils.js#L132-L144

Here if you see the discretely set the first category as `frequentlyUsedEmojis`. Hence to skip the first category with slice I've used `this.unfilteredHeaderIndices[1]`.